lib/puppet/util.rb

```diff
@@ -219,10 +219,11 @@ def benchmark(*args)
                args.pop
              end
 
-    raise Puppet::DevError, "Failed to provide level to :benchmark" unless level
+    #TRANSLATORS 'level' and ':benchmark' should not be translated
+    raise Puppet::DevError, _("Failed to provide level to :benchmark") unless level
```

Comment:
I'm not 100% sure on this, it was just a thought. I'm doubtful about level now that I read it again, but `:benchmark` does seem like something that shouldn't get translated.... I dunno....

Reply:
I would let level be translated, but I agree about no on benchmark.